src/LiquidCrystal.h

```diff
@@ -82,12 +82,22 @@ class LiquidCrystal : public Print {
   void setCursor(uint8_t, uint8_t); 
   virtual size_t write(uint8_t);
   void command(uint8_t);
-  
+  void getCursorPos(int &col, int &row);
+  char getCharAt(uint8_t col, uint8_t row);
+  char getCharAt(uint8_t col, uint8_t row, uint8_t ret);
+  uint8_t getChars(uint8_t col, uint8_t row, char* buffer, uint8_t length);
+  void deleteLast();
+  void deleteAt(uint8_t col, uint8_t row);
+
   using Print::write;
 private:
+  uint8_t readBusyFlagAndAddress();
   void send(uint8_t, uint8_t);
+  uint8_t receive(int rs_pin_mode);
```

Comment:
Where is "receive" function defined? Where is its body? I don't see it in LiquidCrystal.cc.

Reply:
It was present in the original PR:
https://github.com/arduino/Arduino/pull/1517/files#diff-b9b372634e96034dfeb5e18c2d39e936R419-R441